newdle/client/.sass-lint.yml

```diff
@@ -0,0 +1,5 @@
+rules:
+  property-sort-order: 0
+  leading-zero:
+  - 2
+  - include: true
```

Comment:
missing trailing linebreak

Reply:
actually, let's remove this config for now, since having just one rule is not very useful and if we start using it, it should be properly integrated...